.github/workflows/main.yml

```diff
@@ -29,17 +29,17 @@ jobs:
           - 27017:27017
     strategy:
       matrix:
-        node-version: [14.x, 16.x]
+        node-version: [16.x, 18.x]
     steps:
     - uses: actions/checkout@v2
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
     - run: |
-        npm install
+        npm install --legacy-peer-deps
         cd test
-        npm install
+        npm install --legacy-peer-deps
```

Comment:
I think this repo has `.npmrc` with this flag set ... (if it doesn't, it should). In which case, we don't need it in the `ci` scripts.

Reply:
Dropped the `--legacy-peer-deps` flag here: https://github.com/digitalbazaar/bedrock-vc-exchanger/pull/10/commits/bce4d8a4f67e66fd6dc9d427aa9d408d24d5dbe6